config/routes.rb

```diff
@@ -2,4 +2,5 @@
   root 'courses#index'
   resources :courses
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :posts, only: [:index]
```

Comment:
why is only the posts index route created? Maybe these can be combined into one 

`resources :courses, :posts`